lib/modules/manager/fvm/extract.ts

```diff
@@ -3,38 +3,43 @@ import { logger } from '../../../logger';
 import { FlutterVersionDatasource } from '../../datasource/flutter-version';
 import type { PackageDependency, PackageFileContent } from '../types';
 
-interface FvmConfig {
-  flutterSdkVersion: string;
-}
-
 export function extractPackageFile(
   content: string,
   packageFile: string,
 ): PackageFileContent | null {
-  let fvmConfig: FvmConfig;
+  let fvmConfig: any;
```

Comment:
no any! use zod Schema

Reply:
Sorry, can you elaborate? I have little experience Typescript and none with zod.

Reply:
Take a look at this implementation

https://github.com/renovatebot/renovate/blob/8fb2a6b9d24ba40e6e3482fcde542b4529dbeae5/lib/modules/manager/ocb/extract.ts

https://github.com/renovatebot/renovate/blob/8fb2a6b9d24ba40e6e3482fcde542b4529dbeae5/lib/modules/manager/ocb/schema.ts

Reply:
We also have some developer documentation for Zod here:

https://github.com/renovatebot/renovate/blob/main/docs/development/zod.md